src/DoctrineMiddleware.php

```diff
@@ -113,6 +113,7 @@ public function setup()
         if (!$this->setupMetadataDriver($config)) {
             throw new \RuntimeException('No Metadata Driver defined');
         }
+        $config->setAutoGenerateProxyClasses($this->getOption('auto_generate_proxies', true) ? true : false);
```

Comment:
I see your point. I've updated the PR and replaced the ternary with double-equals.